src/App.vue

```diff
@@ -354,6 +362,12 @@ header span {
 .md-dialog {
   box-shadow: initial;
 }
+.md-dialog .md-primary {
+  background: #ffffff;
+}
+.md-dialog-confirm > .md-dialog > .md-dialog-actions > button{
+  color: #000 !important;
```

Comment:
can we make more specified selector, add additional class, instead of using important.